src/features/houseHold/permissions/household.ts

```diff
@@ -0,0 +1,8 @@
+import { getHousehold } from "@/global/actions";
+import { auth } from "@/lib/auth";
+
+export async function canMakeChangesToHousehold(householdId: string) {
```

Comment:
osobiście często takie funkcje nazywam assertMożeCośZrobić

finalnie efektem takiego sprawdzenia prawie zawsze jest to że trzeba wyrzucić jakiś błąd etc dlatego w środku takich funkcji mam od razu throw jeśli coś nie gra

przykład: https://github.com/Newbies-Workplace/jeteo/blob/feat/nest-to-next/src/lib/actions/auth.methods.ts